core/evm.go

```diff
@@ -145,10 +148,3 @@ func Transfer(db vm.StateDB, sender, recipient common.Address, amount *uint256.I
 	db.SubBalance(sender, amount)
 	db.AddBalance(recipient, amount)
 }
-
-func getChainConfig(bc ChainContext) *params.ChainConfig {
-	if bc == nil || reflect.ValueOf(bc).IsNil() {
```

Comment:
this will cause panic
![image](https://github.com/bnb-chain/bsc/assets/7995985/29e03efd-cbb8-4b7f-86bb-7c23dc8d9444)